phenotrex/ml/cccv.py

```diff
@@ -24,29 +24,45 @@ class CompleContaCV:
     A class containing all custom completeness/contamination cross-validation functionality.
 
     :param pipeline: target pipeline which describes the vectorization and estimator/classifier used
-    :param scoring_function: Sklearn-like scoring function of crossvalidation. Default: Balanced Accuracy.
+    :param scoring_function: Sklearn-like scoring function of crossvalidation.
+                             Default: Balanced Accuracy.
     :param cv: Number of folds in crossvalidation. Default: 5
     :param comple_steps: number of steps between 0 and 1 (relative completeness) to be simulated
-    :param conta_steps: number of steps between 0 and 1 (relative contamination level) to be simulated
+    :param conta_steps: number of steps between 0 and 1 (relative contamination level)
+                        to be simulated
     :param n_jobs: Number of parallel jobs. Default: -1 (All processors used)
     :param n_replicates: Number of times the crossvalidation is repeated
     :param reduce_features: toggles feature reduction using recursive feature elimination
     :param n_features: minimal number of features to retain (if feature reduction is used)
     :param random_state: An integer random seed or instance of np.random.RandomState
     """
-    def __init__(self, pipeline: Pipeline, scoring_function: Callable = balanced_accuracy_score, cv: int = 5,
-                 comple_steps: int = 20, conta_steps: int = 20,
-                 n_jobs: int = -1, n_replicates: int = 10, random_state: np.random.RandomState = None,
-                 verb: bool = False, reduce_features: bool = False, n_features: int = 10000):
+    def __init__(
+        self,
+        pipeline: Pipeline,
+        scoring_function: Callable = balanced_accuracy_score,
+        cv: int = 5,
+        comple_steps: int = 20,
+        conta_steps: int = 20,
+        n_jobs: int = -1,
+        n_replicates: int = 10,
+        random_state: np.random.RandomState = None,
+        verb: bool = False,
+        reduce_features: bool = False,
+        n_features: int = 10000
+    ):
         self.pipeline = pipeline
         self.cv = cv
         self.scoring_method = scoring_function
         self.logger = get_logger(__name__, verb=verb)
         if comple_steps < 1:
-            self.logger.warn(f"Completeness steps parameter is out of range: {comple_steps}, was set to 1 instead")
+            self.logger.warn(
```

Comment:
```suggestion
            self.logger.warning(
```
logging.warn is deprecated, https://docs.python.org/3/library/logging.html#logging.Logger.warning